doc/content/verification_validation_examples/CAD_based_workflow/CAD_workflow.md

```diff
@@ -0,0 +1,148 @@
+
+!media figures/cad_workflow_transfers.png 
+  id=transfers
+  caption=OpenMC and MOOSE Coupling
+  style=width:80%;display:block;margin-left:auto;margin-right:auto;
+
+## Generating the meshes
+
+The CAD model was first developed in FUSION360 and was imported into Cubit to assign blocks, materials, and side sets and generate the mesh ([volumetric_mesh]). A corresponding DAGMC surface mesh ([dagmc]) was exported directly from the meshed geometry in Cubit (by loading the volumetric meshed geometry in Cubit and exporting a DAGMC surface mesh).
+
+In this example, `tmesh_1.e` ([volumetric_mesh]) is the finite element mesh used in MOOSE on which the heat conduction physics is solved. `tmesh_1.h5m` ([dagmc]) is the DAGMC surface mesh used for particle transport in OpenMC (which bounds the surfaces between different materials). Cardinal also allows for mesh tallying for tallying OpenMC results directly on the mesh overlayed on the OpenMC geometry which `tmesh_1.e` ([volumetric_mesh]) could be used for as well as an unstructured volume mesh. This could be used by changing the tally type and adding a mesh template (`tally_type = mesh`, `mesh_template = tmesh_1.e`) in Cardinal input under Problem.
+
+&nbsp;
+
+!row! style=display:inline-flex;
+!col! small=12 medium=4 large=3
+
+!media figures/cad_workflow_mesh_1.png 
+  id=volumetric_mesh
+  caption=Model volumetric mesh
+  style=width:130%;display:block;margin-left:auto;margin-right:auto;
+
+!col-end!
+
+!col! small=12 medium=4 large=3
+
+!media figures/cad_workflow_d1.png 
+  id=dagmc
+  caption=Model DAGMC mesh
+  style=width:130%;display:block;margin-left:auto;margin-right:auto;
+
+!col-end!
+!row-end!
+
+## OpenMC
+
+The OpenMC input files is as follows:
+
+!listing /test/tests/examples/cad_workflow/model.py language=python
+
+## Cardinal
+
+The Cardinal input files is shown below. The ([MoabSkinner](https://cardinal.cels.anl.gov/source/userobjects/MoabSkinner.html)) was chosen in this model to update the DAGMC geometry from the MOOSE mesh directly ensuring meshes matching which is furhter discussed in[!cite](Eltawila2024PBNC) as well as the use of cell tallies.
+
+!listing /test/tests/examples/cad_workflow/openmc.i
+
+## FENIX (MOOSE) Heat transfer
+
+The FENIX input files is as follows:
+
+!listing /test/tests/examples/cad_workflow/solid.i
+
+## Execution
+
+To generate OpenMC xml files, run:
+
+```
+python model.py
+```
+
+Then to run the coupled calculation:
+
+```
+mpiexec -np 2 fenix-opt -i solid.i --n-threads=2
+```
+
+This will run both MOOSE and OpenMC (with Cardinal) with 2 MPI processes and 2 OpenMP threads per rank. To run the simulation faster, you can increase the parallel processes/threads, or simply decrease the number of particles used in OpenMC. When the simulation has completed, you will have created a number of different output files:
```

Comment:
```suggestion
This will run both MOOSE and OpenMC (with Cardinal) with 2 MPI processes and 2 OpenMP threads per rank. To run the simulation faster, you can increase the parallel processes/threads, or simply decrease the number of particles used in OpenMC (note that this might also decrease accuracy). When the simulation has completed, you will have created a number of different output files:
```